lib/eslint.js

```diff
@@ -657,12 +660,13 @@ module.exports = (function() {
      *      Mostly useful for testing purposes.
      * @returns {Object[]} The results as an array of messages or null if no messages.
      */
-    api.verify = function(text, config, filename, saveState) {
+    api.verify = function(textOrSourceCode, config, filename, saveState) {
 
         var ast,
             shebang,
             ecmaFeatures,
-            ecmaVersion;
+            ecmaVersion,
+            text = (typeof textOrSourceCode === "string") ? textOrSourceCode : null;
```

Comment:
Because `text` is used in a couple other places as well. This was the easiest way to make the change to `SourceCode` without a ton of refactoring. I'll go back in later and optimize.